pkg/webui/console/lib/device-utils.js

```diff
@@ -156,3 +156,7 @@ export const getLastSeen = device => {
     }
   }
 }
+
+export const isUsBand = (band = '') => band.startsWith('US_902_928')
+export const isAuBand = (band = '') => band.startsWith('AU_915_928')
+export const isCnBand = (band = '') => band.startsWith('CN_470_510')
```

Comment:
So what the technical difference is between bands that do support factory preset frequencies and bands that do not support them, is that the former uses a CFList type of Frequencies, and the latter uses a CFList type of ChMask (channel mask). When there is a channel mask, the frequencies aren't configured by frequency in Hertz, but by index. The factory preset frequencies is really the frequencies in Hertz, so it requires bands with a CFList type of Frequencies.

So, maybe

```suggestion
export const hasCFListTypeChMask = (freqPlan = '') => freqPlan.startsWith('US_902_928') || freqPlan.startsWith('AU_915_928') || freqPlan.startsWith('CN_470_510')
```

Maybe?

Do note that frequency plan IDs start with the band ID, but bands can be string compared. If you pass a frequency plan ID in this context, do not call it `band` but `freqPlan` and use `startsWith()`, otherwise keep using `band` and use string comparison.